spec/controllers/submit_source_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe SubmitSourceController do
+  render_views
+
+  specify 'form works' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.